### PR TITLE
Development

### PR DESCRIPTION
**Pull request by carjug**
## What changes

Deploys app code to cloud.gov sandbox from CircleCI. 

## How to test

See details on this PR.

## Screenshots

Screen Shot 2021-11-23 at 1 20 07 PM

## Links

https://app.circleci.com/pipelines/github/HHS/OPRE-OPS/77/workflows/643c1ff4-d2c8-43b1-a045-634a19d4846a/jobs/81

### CODE REVIEW

**Comment by carjug** [comment]
@amymok this is everything from the previous pull request but just merging from `development` to `main`

**Comment by carjug** [comment]
Still need to update the ADR, so I will let you know when that part is ready @amymok

**Comment by amymok** [comment]
@carjug This looks good so far.  Do you plan to update the ADR in this PR or in another PR?

**Comment by carjug** [comment]
@amymok I'll do it in this PR :) coming right up!

**Comment by carjug** [comment]
@amymok doc updated :)

File: .circleci/config.yml
Comment (amymok) [inline .circleci/config.yml:22]: [issue] I don't think this belongs here.  `filters` are for workflow/jobs, so it goes to a level up.  And I think this is not correct either to only look for tags that begins with `v`.
[suggestion] This should be removed.

File: docs/how_we_work/deploy_flow.md
Comment (amymok) [inline docs/how_we_work/deploy_flow.md:58]: [question] I think it runs even before it gets merged, it will run in a PR if it attempts to merge right?  We may want to express that too.

**Review by amymok — CHANGES_REQUESTED** [review]

File: docs/how_we_work/deploy_flow.md
Comment (carjug) [reply docs/how_we_work/deploy_flow.md:58]: No, I don't think so. We're only deploying to the development environment if it gets merged into development, otherwise we would overwrite development with each new PR to the development branch and I don't think we want that, right? @amymok

Comment (carjug) [reply docs/how_we_work/deploy_flow.md:58]: @amymok I made a junk change with a junk PR #237  just to verify the above is true, and you can see the deploy is skipped if the branch is not `development`. LMK if this is not what you expected.

Screen Shot 2021-12-22 at 10 08 36 AM

Comment (amymok) [reply docs/how_we_work/deploy_flow.md:58]: Oh good good.  Thanks for verifying.  Yes, we only want to deploy when it is merged.  But tests will run regardless.

File: .circleci/config.yml
Comment (amymok) [inline .circleci/config.yml:20]: [nitpick]: This should probably be removed too since there's nothing to filter.

**Review by amymok — CHANGES_REQUESTED** [review]

File: .circleci/config.yml
Comment (carjug) [reply .circleci/config.yml:20]: oh whoops! meant to delete that. will do :)

**Review by amymok — APPROVED** [review]
LGTM!

**Pull request merged**
